src/sonic-yang-mgmt/tests/yang-model-tests/yangModelTesting.py

```diff
@@ -100,6 +101,14 @@ def __init__(self, tests, yangDir, jsonFile):
             'ACL_RULE_WRONG_INNER_ETHER_TYPE': {
                 'desc': 'Configure INNER_ETHER_TYPE as 0x080C in ACL_RULE.',
                 'eStr': self.defaultYANGFailure['Pattern']
+            },
+            'INTERFACE_IPPREFIX_PORT_MUST_CONDITION_FALSE': {
```

Comment:
Should we cover VLAN_INTERFACE and LOOPBACK_INTERFACE here too?

Reply:
okie, I will cover 1 example of VLAN and Loopback too. Because must condition is same in all cases, that is the reason I thought of it as duplicate test case.

Reply:
Done